multiversx_sdk/network_providers/transactions.py

```diff
@@ -28,6 +28,8 @@ class ITransaction(Protocol):
     guardian: str
     signature: bytes
     guardian_signature: bytes
+    relayer: str
+    inner_transactions: Sequence
```

Comment:
can be `inner_transactions: Sequence["ITransaction"]`

Reply:
changed.